tests/engines/collections/acceptance/moderation/moderators-test.ts

```diff
@@ -0,0 +1,45 @@
+import { setupMirage } from 'ember-cli-mirage/test-support';
+import { percySnapshot } from 'ember-percy';
+import { module, test } from 'qunit';
+
+import { visit } from 'ember-osf-web/tests/helpers';
+import { setupEngineApplicationTest } from 'ember-osf-web/tests/helpers/engines';
+
+module('Collections | Acceptance | moderation | moderators', hooks => {
+    setupEngineApplicationTest(hooks, 'collections');
+    setupMirage(hooks);
+
+    test('it renders for collection admins', async assert => {
+        server.create('user', 'loggedIn');
+        const primaryCollection = server.create('collection');
+        const provider = server.create('collection-provider', {
+            id: 'studyswap',
+            primaryCollection,
+        }, 'currentUserIsAdmin');
+        server.createList('moderator', 15, { provider });
```

Comment:
Minor: this will run slightly faster if there are only 11 (10?) created moderators rather than 15 without affecting your test scenario.